org.OpenT2T.Sample.SuperPopular.Thermostat/Wink Thermostat/v2/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "opent2t-translator-wink-thermostat",
+  "version": "1.0.0",
+  "description": "Wink Thermostat translator",
+  "author": "OpenT2T",
+  "license": "MIT",
+  "scripts": {
+    "test": "ava"
+  },
+      "ava": {
+      "files": [
+        "tests/*.js",
+        "!**/deviceconfig.js"
+      ]
+    },
+  "dependencies": {
+    "delay": "^1.3.1",
+    "opent2t": "*",
+    "opent2t-onboarding-winkhub": "*",
+    "opent2t-translator-helper-wink": "1.0.0",
+    "q": "1.4.1"
+  },
+  "devDependencies": {
+    "ava": "^0.15.2"
+  },
+  "opent2t": {
+    "interfaces": {
+      "org.OpenT2T.Sample.SuperPopular.Thermostat": {
```

Comment:
>org.OpenT2T.Sample.SuperPopular.Thermostat [](start = 7, length = 42)

This must match the name of the .js file, because it is the name of the interface module. The name of the .xml file doesn't really matter, though I had expected it would also be the same.

So either rename the .js file to org.OpenT2T.Sample.SuperPopular.Thermostat-2.0.js, or change this to "ThermostatInterface". Or change both to just "Thermostat". Personally I'd recommend the latter.